db.go

```diff
@@ -151,6 +150,16 @@ func (self *DB) resp(method string, params []any, res any) (any, error) {
 
 	arg, ok := params[0].(string)
 
+	peeledResponse := res.([]interface{})
+
+	if len(peeledResponse) == 0 {
+		return nil, nil
+	} else if len(peeledResponse) == 1 {
+		return peeledResponse[0], nil
+	} else if len(peeledResponse) > 1 {
+		return peeledResponse, nil
+	}
+
```

Comment:
Hey @TimothyStiles can you explain a little what the purpose of this is? I get what it's doing, but just trying to understand what the intention is...?

Reply:
@tobiemh just some code to tinker around with. It's been removed in the latest push.

Reply:
testing 

https://github.com/TimothyStiles/surrealdb.go/blob/f35effa9728cf9f319b96df8fc8acf0e442d5876/db_test.go#L53-L75

https://github.com/roots/trellis-cli/blob/a389115eb35e3c322e084183dc41497ffd67e7e0/cmd/venv_hook.go#L25-L27